src/toil/cwl/cwltoil.py

```diff
@@ -456,35 +453,36 @@ def __init__(self, referenced_files, basedir, stagedir,
             referenced_files, basedir, stagedir, separateDirs=separateDirs)
 
     def visit(self, obj, stagedir, basedir, copy=False, staged=False):
-        # type: (Dict[Text, Any], Text, Text, bool, bool) -> None
-        tgt = os.path.join(stagedir, obj["basename"])
+        # type: (Dict[str, Any], str, str, bool, bool) -> None
```

Comment:
This looks like but as far as I can tell isn't actually a real type hint. It should be more clear that this is just text, or it should be a real type hint.

Reply:
Yes, I haven't actually added hints in the proper way here.  I'll go through the entire file and add as needed.